gloo/docs/gloo_routing/validation/_index.md

```diff
@@ -0,0 +1,151 @@
+---
+menuTitle: Validation
+title: Config Reporting & Validation
+weight: 11
+description: >
+ (Kubernetes Only) Gloo can be configured to validate configuration before it is applied to the cluster.
+ With validation enabled, any attempt to apply invalid configuration to the cluster will be rejected.
+---
+
+# Motivation
+
+When configuring an API gateway or edge proxy, invalid configuration can quickly lead to bugs, service outages, and 
+security vulnerabilities. 
+
+This document explains features in Gloo designed to prevent invalid configuration from propagating to the 
+data plane (the Gateway Proxies).
+
+# How Gloo Validates Configuration
+
+Validation in Gloo is comprised of a two step process:
+
+1. First, resources are admitted (or rejected) via a [Kubernetes Validating Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/). Configuration options for the webhook live
+in the `settings.gloo.solo.io` custom resource.
+
+2. Once a resource is admitted, Gloo processes it in a batch with all other admitted resources. If any errors are detected 
```

Comment:
You mention "process it in a batch", how is the batch defined? is there a sync time window?

I can imagine some users seeing this and trying to think through transaction consistency, i.e. I need to update 2+ dependent resources, and they need to be processed together for change to be seen as correct.